package.json

```diff
@@ -7,6 +7,7 @@
 	"type": "module",
 	"scripts": {
 		"prettify": "eslint ./src/ --fix && npx prettier -w .",
+		"prettier": "npm run prettify",
```

Comment:
This is confusing. The name prettier lets you think it only runs prettier without eslint. Please make this clearer

Reply:
Well you can come up with other more intuitive names if you want. But yeah its confusing.